src/lib/components/composites/project-components/CreateProjectDialog.svelte

```diff
@@ -37,14 +37,18 @@
         initialPossibleMembers.filter((member) => !membersInput.includes(member.email)),
     );
 
+    let loadingPossibleMembers = $state(true);
+
     onMount(async () => {
         try {
             const currentUser = await backendService.getCurrentUser(Nothing).response;
             const allUsers = await backendService.getAllUsers(Nothing).response;
 
             initialPossibleMembers = allUsers.users.filter((user) => user.id !== currentUser.id);
+            loadingPossibleMembers = false;
         } catch (error) {
             isErrorOnUsersLoading = true;
+            loadingPossibleMembers = false;
             console.error(`Couldn't get users from server (${error})`);
         }
     });
```

Comment:
Just put it after the try-catch block
```suggestion
        } catch (error) {
            isErrorOnUsersLoading = true;
            console.error(`Couldn't get users from server (${error})`);
        }
        loadingPossibleMembers = false;
    });
```

Reply:
Good idea, I changed it.